src/neuroglancer/datasource/dvid/frontend.ts

```diff
@@ -123,10 +122,12 @@ export class VolumeDataInstanceInfo extends DataInstanceInfo {
                     mat4.create(), vec3.multiply(vec3.create(), lowerVoxelBound, voxelSize)),
                 baseVoxelOffset: lowerVoxelBound,
                 upperVoxelBound: vec3.subtract(vec3.create(), upperVoxelBound, lowerVoxelBound),
-                volumeType: this.volumeType, volumeSourceOptions,
+                volumeType: this.volumeType,
+                volumeSourceOptions,
```

Comment:
specify compressedSegmentationBlockSize as either undefined or vec3.fromValues(8, 8, 8) depending on the volume type.